recovery/options.go

```diff
@@ -3,14 +3,16 @@
 
 package grpc_recovery
 
+import "golang.org/x/net/context"
```

Comment:
Note: do we want this or `context`?

Reply:
Yes, compatibility with old Go that does not have yet `context` is the reason.

We can fix that in later attempt once we agree to not support old Go version (I think 1.6 and below)